library/pk_wrap.c

```diff
@@ -761,20 +761,52 @@ static int pk_opaque_can_do( mbedtls_pk_type_t type )
             type == MBEDTLS_PK_ECDSA );
 }
 
-/* Like mbedtls_asn1_write_mpi, but from a buffer */
+/* translate PSA errors to best PK approximation */
+static int pk_err_from_psa( psa_status_t status )
```

Comment:
Shall I move that to the utility PR?

Reply:
I don't know. It's specific to PK (as PSA errors are translated to the closest PK error), but it could still need to be shared between multiple files in the PK module. @AndrzejKurek what do you think? Would this function be useful in one of your PRs or tasks?

Reply:
@hanno-arm Actually I changed my mind while [reviewing another PR](https://github.com/ARMmbed/mbedtls/pull/2159#discussion_r233758471) and now think this should be moved to #2184 :)

Reply:
@mpg Ok, I will do that and ping you once it's done.

Reply:
@mpg Ping, I updated #2184 now.